pkg/adapter/cloudeventingress/adapter.go

```diff
@@ -0,0 +1,99 @@
+/*
+Copyright 2018 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cloudeventingress
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"math/rand"
+	"net/http"
+	"time"
+
+	"github.com/knative/pkg/cloudevents"
+	"github.com/knative/pkg/logging"
+	"go.uber.org/zap"
+)
+
+const (
+	eventType = "dev.knative.cronjob.event"
+)
+
+// Adapter implements the CloudEventsIngress to trigger a Sink.
+type Adapter struct {
+	// Messasge is the data to be posted to the target.
+	Message string
+
+	// SinkURI is the URI messages will be forwarded on to.
+	SinkURI string
+}
+
+func (a *Adapter) Start(ctx context.Context) error {
```

Comment:
This looks like it sends a single event once, is that what it is supposed to be doing?

Reply:
Changed. Please review the new version.